bigbluebutton-html5/imports/ui/services/api/index.js

```diff
@@ -1,6 +1,24 @@
 import Auth from '/imports/ui/services/auth';
 import { check } from 'meteor/check';
 import { notify } from '/imports/ui/services/notification';
+import logger from '/imports/startup/client/logger';
+
+export function log(type = 'error', message, ...args) {
```

Comment:
Wasn't this log function just the old way we were logging when it was only sent to the server? Does it even need to exist anymore?